src/Html.php

```diff
@@ -77,61 +80,63 @@ final class Html
         'rel',
         'media',
     ];
+
     /**
-     * @var array list of tag attributes that should be specially handled when their values are of array type.
-     * In particular, if the value of the `data` attribute is `['name' => 'xyz', 'age' => 13]`, two attributes
-     * will be generated instead of one: `data-name="xyz" data-age="13"`.
+     * @var array list of tag attributes that should be  specially  handled  when  their  values  are  of  array  type.
+     * In particular, if the value of the `data` attribute is `['name' => 'xyz', 'age' => 13]`, two attributes will  be
+     * generated instead of one: `data-name="xyz" data-age="13"`.
+     * @since 2.0.3
      */
     public static $dataAttributes = ['data', 'data-ng', 'ng'];
 
 
     /**
-     * Encodes special characters into HTML entities.
-     * The {@see \yii\base\Application::charset|application charset} will be used for encoding.
-     * @param string $content the content to be encoded
-     * @param bool $doubleEncode whether to encode HTML entities in `$content`. If false,
-     * HTML entities in `$content` will not be further encoded.
+     * Encodes special characters into HTML entities. The [[\yii\base\Application::charset|application  charset]]  will
+     * be used for encoding.
+     * @param string|null $content the content to be encoded
+     * @param bool $doubleEncode whether to encode HTML entities in `$content`. If false, HTML  entities  in  `$content`
+     * will not be further encoded.
      * @return string the encoded content
      * @see decode()
      * @see http://www.php.net/manual/en/function.htmlspecialchars.php
      */
-    public static function encode($content, $doubleEncode = true): string
+    public static function encode($content, bool $doubleEncode = true): string
     {
-        return htmlspecialchars($content, ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML5, ini_get('default_charset'), $doubleEncode);
+        return htmlspecialchars($content, ENT_QUOTES | ENT_SUBSTITUTE, ini_get('default_charset'), $doubleEncode);
```

Comment:
Why removing HTML5?

Reply:
I did not delete it was updated in Yii / Core / Helpers, from there comes that update.

https://github.com/yiisoft/yii-core/blob/011de4ceb1ffa702550a494e60af182f9b30c6a1/src/helpers/BaseHtml.php#L111

Reply:
Well, the initial code here is older than the one that was in yii-core...

Reply:
No problem I will add ENT_HTML5, and review all observations well regarding documentation.

Reply:
Checking the documentation, i see that if I add ENT_HTML5 | ENT_QUOTES will return `&apos;` and the one in the test is with `&#039;` in yii2 it is not enabled either, so if you add it you have to change the test a `&apos;`.

https://www.php.net/htmlspecialchars

Then I make the change, and the documentation is ready.

Reply:
Yes. Let's adjust tests/